open-broker-common/src/main/java/de/fraunhofer/iais/eis/ids/broker/core/common/persistence/SelfDescriptionPersistenceAndIndexing.java

```diff
@@ -334,19 +339,24 @@ public URI updated(InfrastructureComponent infrastructureComponent) throws IOExc
         //If the connector was passive before, the document was deleted from the index, so we need to recreate it
         if (wasActive) { //Connector exists in index - update it
             try {
+            	logger.info("Updating "+infrastructureComponent+" in index");
                 indexing.update(infrastructureComponent);
             } catch (Exception e) {
                 if (e.getMessage().contains("document_missing_exception")) {
-                    indexing.add(infrastructureComponent);
+                	logger.info("Adding "+infrastructureComponent+" in index");
+                	indexing.add(infrastructureComponent);
                 } else {
                     logger.error("ElasticsearchStatusException caught with message " + e.getMessage());
                     throw new RejectMessageException(RejectionReason.INTERNAL_RECIPIENT_ERROR, e);
                 }
             }
         } else { //Connector does not exist in index - create it
-            indexing.add(infrastructureComponent);
+        	logger.info("Creating new "+ infrastructureComponent+" as it was not available earlier.");
+        	indexing.add(infrastructureComponent);
         }
         //return the (rewritten) URI of the infrastructure component
+        logger.info("The rewritten URI of the " +infrastructureComponent+" " + infrastructureComponent.getId().toString());
```

Comment:
This line makes no sense. Logging "infrastructureComponent" will give you some Java object ID